features/upgrade/routing/upgrade.feature

```diff
@@ -276,9 +276,12 @@ Feature: Routing and DNS related scenarios
     Then the step should succeed
     When I use the "ocp45955" project
     Given I obtain test data file "routing/web-server-rc.yaml"
+    Given I wait up to 180 seconds for the steps to pass:
```

Comment:
@melvinjoseph86 I'm curious why creating a web-server can take up to 180 seconds to finish.  That doesn't sound right to me and perhaps having this wait masks the real culprit.

Reply:
@pruan-rht i reduced it to 30 seconds
I was trying to avoid some failures which may caused due to timing issue.